data/bash-completion/fwupdtool

```diff
@@ -22,6 +22,7 @@ _fwupdtool_cmd_list=(
 	'get-history'
 	'get-plugins'
 	'get-remotes'
+	'get-report-metadata'
```

Comment:
should this be get-device-report-metadata maybe since it's focused on one device?

Reply:
Yeah that's exactly my worry.

Reply:
`get-device-report-metadata` breaks the `get-dev<tab>` autocomplete for me -- what about just `device-metadata` or `device-report-metadata`? Or `report-metadata-device` perhaps?

Reply:
Or how about you just build a report for all metadata instead of just one device?

Reply:
Actually that's not insane; most of the time I'm using --plugins to filter anyway.

Reply:
Something like ^^?